src/fpm_environment.f90

```diff
@@ -157,13 +161,28 @@ subroutine run(cmd,echo,exitstat)
             exitstat = stat
         else
             if (stat /= 0) then
-                print *, 'Command failed'
-                error stop
+                call fpm_stop(stopmsg='Command_failed')
             end if
         end if
 
     end subroutine run
 
+    !> if verbose mode perform an ERROR STOP else perform a STOP
+    subroutine fpm_stop(stopcode,stopmsg)
+    implicit none
+    integer,intent(in),optional          :: stopcode
+    character(len=*),intent(in),optional :: stopmsg
+    integer                              :: local_code
+    integer                              :: ios
+       if(present(stopcode))then;local_code=stopcode;else;local_code=0;endif
+       if(present(stopmsg)) write(stderr,'(a)',iostat=ios)esc('<r><bo>'//stopmsg)
+       if(config%verbose)then
+          error stop local_code
```

Comment:
I recently removed usage of `error stop` in the backend and I don't think we really need it in _fpm_. The backtrace isn't good or helpful for user experience. If we're purposely stopping execution better to just ensure meaningful error messages IMO. Perhaps add an optional argument to describe the error location (_e.g._ `call fpm_stop(location="fpm_filesystem::mkdir")`)